src/components/list/list.css

```diff
@@ -25,28 +25,30 @@
   border: none;
 }
 
-.btn-sidebar > img {
+.sidebar__btn > img {
   margin-top: 5px;
   margin-bottom: 5px;
 }
-.iconAudio {
+.sidebar__iconAudio {
```

Comment:
mabye better with .sidebar__icon--audio
because its another version of icon?

Reply:
yes!!! good point! 🤩
thank you :)